Makefile.common

```diff
@@ -168,13 +168,8 @@ PLUGINSCMO:=$(LTACCMO) $(OMEGACMO) $(MICROMEGACMO) \
         $(FUNINDCMO) $(NSATZCMO) $(SYNTAXCMO) \
 	$(DERIVECMO) $(SSRMATCHINGCMO) $(SSRCMO) $(LTAC2CMO) $(ZIFYCMO)
 
-ifeq ($(HASNATDYNLINK)-$(BEST),false-opt)
```

Comment:
What's the matter with this change?

Reply:
This is the code I use to debug plugins. I commited it by mistake (now removed).

Note in passing that this shows that something is wrong with the makefile which fails on the Ltac2 plugin when dynlink is not available.